db/migrate/20150729170936_add_items.rb

```diff
@@ -0,0 +1,12 @@
+class AddItems < ActiveRecord::Migration
+  def change
+    create_table :items do |t|
+      t.string :title, null: false
+      t.string :description, null: false
+      t.string :artist, null: false
```

Comment:
As in having an "artist" user put in their own info for their designs in our system? 
Wouldn't it be easier if we ("admin" user) could do it at the time of creation?